src/serialise/ahoXml/generate.ts

```diff
@@ -216,12 +187,8 @@ const mapNextResultSourceOrganisation = (
   return mapAhoOrgUnitToXml(ou)
 }
 
-const mapAhoResultsToXml = (
-  results: Result[],
-  exceptions: Exception[] | undefined,
-  offenceIndex: number
-): Br7Result[] =>
-  results.map((result, resultIndex) => ({
+const mapAhoResultsToXml = (results: Result[]): Br7Result[] =>
+  results.map((result) => ({
     "ds:CJSresultCode": text(result.CJSresultCode.toString()),
```

Comment:
@bjpirt Is the incorrect casing intentional here? It's the same in the aho schema too